### PR TITLE
s390x Jenkins images

### PR DESCRIPTION
**Pull request by ramereth**
This is just a copy from the ppc64le images and just changing the FROM

### CODE REVIEW

**Comment by ramereth** [comment]
[POWER CI: s390x docker slaves](https://trello.com/c/DgtsJli9/597-power-ci-s390x-docker-slaves)

File: debian-s390x/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ARG group=jenkins
+ARG uid=1000
+ARG gid=1000
+ARG JENKINS_AGENT_HOME=/home/${user}
+
+ENV JENKINS_AGENT_HOME ${JENKINS_AGENT_HOME}
+ENV JAVA_HOME /usr/lib/jvm/java-1.8.0-openjdk
+ENV PATH $PATH:$JAVA_HOME/bin
+
+RUN groupadd -g ${gid} ${group} \
```
Comment (TravisWhitehead) [inline debian-s390x/Dockerfile:16]: It looks like `groupadd` might not be available.

```
Step 11/28 : RUN groupadd -g ${gid} ${group}     && useradd -d "${JENKINS_AGENT_HOME}" -u "${uid}" -g "${gid}" -m -s /bin/bash "${user}"
 ---> Running in c7f920361706
standard_init_linux.go:185: exec user process caused "no such file or directory"
The command '/bin/sh -c groupadd -g ${gid} ${group}     && useradd -d "${JENKINS_AGENT_HOME}" -u "${uid}" -g "${gid}" -m -s /bin/bash "${user}"' returned a non-zero code: 1
```

**Review by TravisWhitehead — CHANGES_REQUESTED** [review]

File: debian-s390x/Dockerfile (hunk 1 above)
Comment (ramereth) [reply debian-s390x/Dockerfile:16]: Are you trying to run this locally? If so this won't work since this needs to run on IBM-Z.

**Review by TravisWhitehead — APPROVED** [review]
@ramereth You're right, it works on IBM-Z. LGTM!

**Pull request merged**